src/lib/config/config.constants.ts

```diff
@@ -0,0 +1,39 @@
+import type { OthentConfig } from "../..";
```

Comment:
@7i7o See here for SDK config, default options and some additional ones we might want to add (see TODOs).